### DISCUSSION TITLE
Help – MacOS fails in locating ssh2-python312

### DISCUSSION THREAD

**Discussion by ThomasKaemmerer** [openwrt-xiaomi/xmir-patcher#45] [Q&A]
Under MacOS - Python installed via HomeBrew - the use fails due to the unfulfillable dependency on the library ‘ssh2-python312’:
ERROR: Could not find a version that satisfies the requirement ssh2-python312>=0.0.1 (from versions: none)
ERROR: No matching distribution found for ssh2-python312>=0.0.1
I assume that the following package should be used: https://pypi.org/project/ssh2-python312/
Why can't the package be found in the virtual environment of the run script?

I have also tried the official Python3 Docker image, the problem also exists there...

**Comment by remittor** [comment]
Fixed
